src/quo2/components/tags/network_tags/view.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.tags.network-tags.view
+  (:require [quo2.components.tags.base-tag :as base-tag]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.components.tags.network-tags.style :as style]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(defn network-tags
+  [_ _]
+  (fn [{:keys [title networks status]}]
+    [rn/view
+     {:style (style/network-tags-container status)}
+     [base-tag/base-tag
+      {:size 32
+       :type :permission}
+      [rn/view {:style (style/network-tag-view)}
+       [preview-list/preview-list {:type :network :size 32} networks]
```

Comment:
I think the size used in the designs for the tokens is 16x16? https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=22567-204965&mode=design&t=vhA1JNNQuKcyKCKC-0

Reply:
Done!